code/game/turfs/simulated/mineral.dm

```diff
@@ -11,12 +11,8 @@
 /turf/simulated/floor/mineral/plasma
 	name = "plasma floor"
 	icon_state = "plasma"
-	floor_tile = /obj/item/stack/tile/mineral/plasma
```

Comment:
I feel like the New() call for all these should just be getFromPool(floor_tile, null) at the base level since every single variation of this New() is doing the same thing